project_files/Xcode/hector.xcodeproj/xcshareddata/xcschemes/ssp245.xcscheme

```diff
@@ -53,7 +53,7 @@
       </BuildableProductRunnable>
       <CommandLineArguments>
          <CommandLineArgument
-            argument = "./inst/input/hector_ssp245.ini"
+            argument = "/Users/dorh012/Documents/Hector-WD/hector/inst/input/hector_ssp245.ini"
```

Comment:
This absolute path on @kdorheim 's machine should be changed to a relative one.

Reply:
AH! That shouldn't have been committed! Sloppy on my part!